app/assets/javascripts/tabs.js

```diff
@@ -145,6 +145,8 @@
 
 window.addEventListener("load", function(event) {
     var errorSummary = document.getElementById("error-summary-display");
+    var errorSummarylinkHref = document.querySelector("[href='#manage']");
     if(errorSummary) errorSummary.focus();
+    if(errorSummarylinkHref) errorSummarylinkHref.setAttribute('href', '#dd');
 });
```

Comment:
The code to focus the error summary makes sense to any reader of the code, but it's not clear the reason for manipulating the `linkHref` fragment - it looks very specific to one page's error summary. 

Can it be dealt with on the page that needs it instead without resorting to javascript? 

Such a javascript workaround will be a surprise to maintainers of the codebase who'd normally expect to understand the behaviour from just looking at the one page - i.e. this sort of javascript workaround can be a kind of spooky action at a distance - what if someone who wasn't aware of this javascript were to change the `#manage` fragment to something else?